src/skins/vector/css/matrix-react-sdk/structures/_FilePanel.scss

```diff
@@ -15,13 +15,8 @@ limitations under the License.
 */
 
 .mx_FilePanel {
-    -webkit-box-ordinal-group: 2;
```

Comment:
it'd probably be good to keep refactoring the CSS separate from cleaning it up as much as possible, if nothing else to give an easier way to point blame if it starts doing something weird.